devops/azure-pipelines.yml

```diff
@@ -0,0 +1,48 @@
+# This pipeline relies on the analysis.yml template located in the
+# repository resource descibed below
+resources:
+  repositories:
+  - repository: templates
+    type: github
+    endpoint: Team1-Project3
+    name: 1053-August-Duet-Project-Registry/project-registry-devops
+    ref: refs/heads/feature-templates
+
+# This pipeline will trigger on commits or pull-requests made to the branch-dev
+# or main branches.  Note that static analysis with SonarCloud is only performed
+# on "long-lived" branches, which are designated as 'main' and any branch name
+# prefixed with 'branch-'.
+trigger:
+- branch-dev
```

Comment:
the `branch-` prefix you've adopted is a bit unusual, since the ones without this prefix are branches too. but i won't complain

Reply:
this naming convention correlates with sonarCloud's default for designating longer-lived branches.